server/__init__.py

```diff
@@ -41,6 +41,7 @@
     "education",
     "demographics",
     "housing",
+    "climate",
     "environment",
```

Comment:
climate and environment seem very similar sounding.

seems like environment only contains disasters now?

Reply:
environment has energy and co2 emission now. I am thinking split it into "energy" and "environment" afterwords and adding more energy charts?

Reply:
That sounds better.  Also, it might be nicer to line up these sections with verticals on SV hierarchy: https://screenshot.googleplex.com/9ijDDViorHF7AZJ